src/main/java/org/pesmypetcare/webservice/dao/usermanager/UserDaoImpl.java

```diff
@@ -81,6 +87,7 @@ public void createUser(String uid, UserEntity userEntity) throws DatabaseAccessE
             batch.set(users.document(uid), userEntity);
             try {
                 batch.commit().get();
+                ((UserMedalDaoImpl)userMedalDao).createAllUserMedals(username);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['typecast' is not followed by whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5641243)